docs/introduction.md

```diff
@@ -67,83 +96,207 @@ Flow is based upon the following set of rules:
 As opposed to other node roles, consensus nodes deal with subjective problems, where there is no single correct answer.
 Instead, one answer must be selected through mutual agreement, which is why it is critical for consensus nodes to be numerous and decentralized, and less so for other nodes.
 This design increases decentralization by allowing for higher levels of participation in consensus by individuals with suitable consumer hardware on home internet connections.
 
 When a consensus node receives a guaranteed collection of transactions, it has to run its consensus algorithm to reach an agreement with other nodes over the set of collections to be included in the next block.
-A block of the ordered collection that has undergone the complete consensus algorithm is called a _finalized block_.
+A block of the ordered collections that has undergone the complete consensus algorithm is called a **finalized block**.
 A block specifies the included collections as well as the other inputs (randomness seed, etc.) which are required to execute the computation.
 It is worth noting that a block in Flow does not include the resulting execution state of the block execution.
 
 In order for consensus nodes to seal blocks, they must commit to the execution result of a block after it is executed and verified.
 
-##### Collection Nodes
+#### Block Formation
 
-For the sake of load-balancing, redundancy and [Byzantine resilience](#byzantine-fault), the collection nodes are [staked](#staking) equally and randomly positioned into clusters of roughly identical size.
-At the beginning of an epoch, each collection node is randomly assigned to exactly one cluster.
-Each cluster of collection nodes acts as a gateway to Flow, from the external world.
-This clustering mechanism avoids heterogeneous systems where a collection node with better service would be getting all the traffic and end up reducing the decentralization of the whole system as well as starving out other collections.
+Block formation is a continuous process performed by the consensus nodes to form new blocks. Block formation has several purposes:
+
+1. Including newly submitted guaranteed collections and reaching aggreement over the order of them
+2. Provide a measure of time by continuously publishing blocks (determining the length of an epoch)
+3. Publish block seals for previous blocks of the chain. The block is ready to be sealed when
+    1. It has been finalized by the consensus nodes
+    2. It has been executed by the execution nodes
+    3. The execution results are approved by the verification nodes
+4. Publishing slashing challenges and respective adjudications results (rulings)
+5. Publishing protocol state updates
+    - staking, unstaking and slashing
+    - whenever a nodes stake changes, consensus nodes include that update in the next block
+6. Providing a state of randomness
```

Comment:
I don't know if it's worded like this in the Flow docs, but it doesn't make sense. It's a `seed` for randomness.